### PR TITLE
Modify multi_nic_hotplug for s390x

### PR DESCRIPTION
**Pull request by fbq815**
Virtual_network: modify the annotation of cfg file

ID: 1942866

Signed-off-by: Boqiao Fu <bfu@redhat.com>

### CODE REVIEW

**Comment by fbq815** [comment]
this patch fixed the error caused by https://github.com/autotest/tp-qemu/pull/2727

**Comment by fbq815** [comment]
Test result:
JOB ID     : fecca85310327efa4a81e78386d925bd85aa20c3
JOB LOG    : /home/bfu/kar/workspace/job-results/job-2021-05-18T07.05-fecca85/job.log
 (1/3) Host_RHEL.m8.u5.product_av.nographic.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.s390x.io-github-autotest-qemu.migrate.tcp.s390-virtio: PASS (63.21 s)
 (2/3) Host_RHEL.m8.u5.product_av.nographic.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.s390x.io-github-autotest-qemu.migrate.tcp.with_filter_off.with_post_copy.s390-virtio: PASS (51.12 s)
 (3/3) Host_RHEL.m8.u5.product_av.nographic.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.s390x.io-github-autotest-qemu.migrate.tcp.with_filter_off.with_multifd.s390-virtio: PASS (60.60 s)
RESULTS    : PASS 3 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /home/bfu/kar/workspace/job-results/job-2021-05-18T07.05-fecca85/results.html
JOB TIME   : 175.77 s
JOB ID     : 0ac68c85177dfb8de7bae7fc95e8c71fabfead07
JOB LOG    : /home/bfu/kar/workspace/job-results/job-2021-05-18T06.57-0ac68c8/job.log
 (1/2) Host_RHEL.m8.u5.product_av.nographic.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.s390x.io-github-autotest-qemu.multi_nics_hotplug.nic_virtio.s390-virtio: PASS (225.72 s)
 (2/2) Host_RHEL.m8.u5.product_av.nographic.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.s390x.io-github-autotest-qemu.multi_nics_hotplug.sameid.s390-virtio: PASS (127.33 s)
RESULTS    : PASS 2 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /home/bfu/kar/workspace/job-results/job-2021-05-18T06.57-0ac68c8/results.html
JOB TIME   : 354.60 s

**Comment by fbq815** [comment]
@PaulYuuu please help me review this patch

**Comment by PaulYuuu** [comment]
cc @yanan-fu

File: qemu/tests/cfg/multi_nics_hotplug.cfg [hunk 1]
```diff
@@ -26,9 +26,8 @@
             nics = ""
             extra_params += "-net none"
             s390x:
-            ''' virtio-net-ccw is not a kind of pci device, but
-            more convinient to use pci_model here
-            '''
+            # virtio-net-ccw is not a kind of pci device, but more
+            # convenient to use pci_model here
```
Comment (yanan-fu) [inline qemu/tests/cfg/multi_nics_hotplug.cfg:30]: It is better to align with `pci_model = virtio-net-ccw` , or move to the front of `s390x:`.

**Comment by clebergnu** [comment]
Apart from the suggestion from @yanan-fu, this LGTM.  Let's try to get this is ASAP, because it's breaking every single new (default) deployment of Avocado-VT (and its CI too).

File: qemu/tests/cfg/multi_nics_hotplug.cfg (hunk 1 above)
Comment (fbq815) [reply qemu/tests/cfg/multi_nics_hotplug.cfg:29]: updated

**Comment by yanan-fu** [comment]
Thanks @fbq815 

Test with the latest patch. can parse test cases normally now.
```
# git am 2822.patch 
Applying: Modify multi_nic_hotplug for s390x
```

**Pull request merged**